lang/en/docs/tutorials/materials/specific/defect_point_adatom_island_mos2_pt.md

```diff
@@ -0,0 +1,173 @@
+---
+# YAML header
+render_macros: true
+---
+
+# Pt Adatom Island on MoS2
+
+## Introduction
+
+This tutorial demonstrates how to create a platinum island on MoS2 by sequentially adding Pt adatoms, following the methodology described in the literature.
+
+!!!note "Manuscript"
+    Saidi, W. A.
+    "Density Functional Theory Study of Nucleation and Growth of Pt Nanoparticles on MoS2(001) Surface"
+    Crystal Growth & Design, 15(2), 642–652. (2015)
+    [DOI: 10.1021/cg5013395](https://doi.org/10.1021/cg5013395){:target='_blank'}.
+
+We will recreate the Pt island structure shown in Figure 4b:
+
+![Pt Island on MoS2](/images/tutorials/materials/defects/defect_point_adatom_island_mos2_pt/0-figure-from-manuscript.webp "Pt island formation on MoS2")
+
+## 1. Create MoS2 Substrate
+
+### 1.1. Load Base Material
+
+Navigate to [Materials Designer](../../../materials-designer/overview.md) and import the MoS2 2D material from [Standata](../../../materials-designer/header-menu/input-output/standata-import.md).
+
+### 1.2. Launch JupyterLite Session
+
+Select the "Advanced > [JupyterLite Transformation](../../../materials-designer/header-menu/advanced/jupyterlite-dialog.md)" menu item to launch the JupyterLite environment.
+
+### 1.3. Open `create_adatom_defect.ipynb` Notebook
+
+Find and open the `create_adatom_defect.ipynb` notebook. We'll use this notebook multiple times to add Pt adatoms sequentially.
+
+## 2. Add Pt Adatoms
+
+We'll add four Pt atoms sequentially to form the island. Each atom needs specific parameters to achieve the correct structure.
```

Comment:
The sequential process is somewhat inconvenient - can we just add them all at once?